docs/usage-with-routing.stories.mdx

```diff
@@ -23,6 +23,7 @@ Note the use of the `useHistory` hook from `react-router-dom`.
 import React, { useCallback } from "react";
 import Button from "carbon-react/lib/components/button";
 import Link from "carbon-react/lib/components/link";
+import {Tabs, Tab} from "carbon-react/lib/components/Tabs";
```

Comment:
The import path should have `/tabs` with a lower case `t` and gaps in this bit `{ Tabs, Tab }`

Reply:
I recommend setting up prettier to run on save in vscode (or whatever editor you use)